bottomline_recordandreplay/manifest.json

```diff
@@ -0,0 +1,73 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "d87fbcfa-71db-4d62-8264-5d88ba2338ce",
+  "app_id": "bottomline-recordandreplay",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitor your 3270/5250 Mainframe users and Mainframe resources using network traffic.",
+    "title": "Bottomline's Record and Replay: Mainframe Integration",
+    "media": [
+      {
+        "media_type": "video",
+        "caption": "Record and Replay Overview",
+        "image_url": "https://bcove.video/3Frep0X"
+      },
+      {
+        "media_type": "image",
+        "caption": "Mainframe Record and Replay User Session",
+        "image_url": "images/mainframe_replay.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Mainframe Record and Replay Dashboard",
+        "image_url": "images/bt_dashboard.png"
```

Comment:
There are four images uploaded but only two that are referenced.

Reply:
fixed